src/typechecker.erl

```diff
@@ -2588,10 +2588,11 @@ do_type_check_expr_in(Env, ResTy, {'receive', _, Clauses}) ->
     {Env1, Cs} = check_clauses(Env, [type(any)], ResTy, Clauses, capture_vars),
     {Env1, Cs};
 do_type_check_expr_in(Env, ResTy, {'receive', _, Clauses, After, Block}) ->
+    VB = Env#env.venv,
```

Comment:
VB is short for var binds, i.e. variables bound in a the checked piece of code.

I think VEnv is a better name for this variable.